lms/static/sass/shared-v2/_components.scss

```diff
@@ -150,6 +150,13 @@
   }
 }
 
+.section {
+    .icon {
+        width: 20px;
+        text-align: center;
+    }
+}
+
```

Comment:
Hi! one small nit here is that it would be nice to add a bit of space between these two tools! No other thoughts on my end, the course updates tab could maybe eventually use some polish / detailed review but Liz is planning to do a review all in one shot later to determine which if any of the changes are necessary. Thanks!